code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -256,7 +256,34 @@
 		layer = OBJ_LAYER
 		plane = OBJ_PLANE
 
+/obj/structure/bed/chair/office/relaymove(var/mob/living/user, direction)
+	if(user.isStunned() || user.isUnconscious() || !user.has_limbs)
```

Comment:
user.isIncapacitated() || !user.has_limbs

Reply:
No, you can still scoot while cuffed

Reply:
let's not

Reply:
please god no

Reply:
why

Reply:
My reasoning is that it'd be hilarious under certain circumstances, but I don't think it would really ever be an inconvenience for antags or otherwise

All the office chairs are rare, indoors and in public places, you can usually find something else to buckle them to in the same room or just wrench it into a regular chair, it's slow as all fuck, and they're interrupting their resist

This isn't some hippie stun resist people

Reply:
Also legcuffs being more commonplace when

Reply:
WYCI

Reply:
are you really telling me something that can be made with a few sheets of metal is rare

Reply:
Rare in the map and if you're making them from sheets you can just make a normal chair but what the fuck ever